.github/workflows/changelog-summary.yml

```diff
@@ -0,0 +1,42 @@
+name: Changelog Summmary
+
+on:
+  push:
+    branches:
+      - staging
+      - update/ci
+
+permissions:
+  contents: read
+  pull-requests: write
```

Comment:
BTW, do we really need this permission?

Reply:
I mean, maybe `read` is enough?

Reply:
We are changing labels on PR. I think that's why we need write.